dali/pipeline/pipeline.h

```diff
@@ -302,7 +302,10 @@ class DLL_PUBLIC Pipeline {
       std::seed_seq ss{seed};
       ss.generate(seed_.begin(), seed_.end());
     } else {
-      std::seed_seq ss{time(0)};
+      using Chrono = std::chrono::high_resolution_clock;
+      auto init_value = std::chrono::duration_cast<std::chrono::nanoseconds>
+          (Chrono::now().time_since_epoch()).count();
```

Comment:
```suggestion
    auto init_value = std::chrono::high_resolution_clock::now().time_since_epoch().count();
```
High resolution clock already has appropriate resolution; no need for a duration cast here.

Reply:
Done